common/src/chain/block/block_reward.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://spdx.org/licenses/MIT
```

Comment:
We've changed the url for this

Reply:
https://github.com/mintlayer/mintlayer-core/pull/351#discussion_r940640078